syntax/forth.vim

```diff
@@ -15,8 +15,12 @@ let s:cpo_save = &cpo
 set cpo&vim
 
 " Synchronization method
-syn sync ccomment
-syn sync maxlines=200
+if exists("forth_minlines")
+    let b:forth_minlines=forth_minlines
+else
+    let b:forth_minlines=50
+endif
+exec "syn sync minlines=" . b:forth_minlines
```

Comment:
As a stylistic option you could use the following if you prefer.

```vim
exe "syn sync minlines=" .. get(g:, "forth_minlines", 50)
````

Reply:
that is a nice improvement, thanks!